app/routes/provider.js

```diff
@@ -8,15 +9,39 @@ import Ember from 'ember';
 /**
  * @class Provider Route Handler
  */
-export default Ember.Route.extend({
+export default Ember.Route.extend(queryParamsMixin, {
```

Comment:
This would be better suited as the `index` route, the counts will only be seen on this page, correct?

Reply:
we need it for this page only